src/test_driver/linux-cirque/README.md

```diff
@@ -79,3 +79,27 @@ Or
 LOG_DIR=/some/log/directory scripts/tests/cirque_tests.sh run_all_tests
 LOG_DIR=/some/log/directory scripts/tests/cirque_tests.sh run_test OnOffClusterTest
 ```
+
+## Setup test topology only
+
+You can run a ManualTest to setup test topology only:
+
+```
+scripts/tests/cirque_tests.sh run_test ManualTest
+```
+
+It will print the container id in log, you can attach to it or execute command
```

Comment:
Can we print the commands we can run? for example, im test, print wrapped command?